src/components/StakingPage/StakingPanel/LockPeriodOptions.tsx/index.tsx

```diff
@@ -0,0 +1,135 @@
+/** @jsx jsx */
+import { jsx } from 'theme-ui';
+import { memo } from 'react';

```

Comment:
Combine imports from react

Reply:
jsx from `theme-ui` has `sx` attributes, if using react `jsx` for `JSX.Element` would have types issue

Reply:
I means the other import above:

import { useState } from 'react';
import { useMemo } from 'react';
import { useEffect } from 'react';
import { useRef } from 'react';

Reply:
i see, vscode autoimport bugs, will fix